forest-node/ansible/ansible.cfg

```diff
@@ -0,0 +1,8 @@
+[defaults]
+host_key_checking = false
+inventory = hosts
+
+remote_user = chainsafe
+private_key_file = ~/id_rsa
```

Comment:
What is this private key exactly? Why does ansible need it? Why is it `id_rsa`? On my machine, it's not the case.

Reply:
The private key is needed for ansible to successfully `ssh` into the server. The `id_rsa` is the conventional name given to private keys when they are created but for sure it is different in your machine because you put a custom name upon creation. 

If you use the terminal suggestion, it will be `id_rsa` and `id_rsa.pub`

Reply:
So the user will get a random error from Ansible in case he doesn't use the default name. It's a bit harsh. I recommend putting it as a variable. Ultimately, when I run this on my machine I shouldn't have any changes in the tracked source files. Do you agree?

Reply:
I don't think I agree because in the requirements step, it is necessary to `ssh keygen` which means before even attempting to run the playbook, the files should have been created if the user follows the docs